perspective.el

```diff
@@ -1227,10 +1234,23 @@ Each perspective's buffer list and window layout are also
 restored."
   (interactive (list
                 (read-file-name "Restore perspective state from file: "
-                                persp-state-default-file
-                                persp-state-default-file)))
-  (unless (file-exists-p file)
-    (error "File not found"))
+                                (if (and
+                                     (bound-and-true-p persp-state-default-file)
+                                     (file-directory-p (file-name-directory persp-state-default-file)))
+                                    (file-name-directory persp-state-default-file)
+                                  user-emacs-directory)
+                                (if (and
+                                     (bound-and-true-p persp-state-default-file)
+                                     (file-readable-p persp-state-default-file))
+                                    (file-name-nondirectory persp-state-default-file)
+                                  "perspective.persist.el"))))
```

Comment:
Same comments on this as for `persp-state-save`. The call for `read-file-name` should not change, though you may want to add the `interactive?` parameter and corresponding `t` as for (unpatched) `persp-state-save`.